ext/standard/tests/file/bug28790.phpt

```diff
@@ -0,0 +1,77 @@
+--TEST--
+Bug #28790: Add php.ini option to disable stat cache
+--CREDITS--
+jnoll <jnoll [at] prim [dot] hu>
+Kevin Lyda <kevin [at] lyda [dot] ie>
+--FILE--
+<?php
+
+    $inifile = __DIR__.DIRECTORY_SEPARATOR.'bug28790.ini';
+    $php = '"'.getenv('TEST_PHP_EXECUTABLE').'"';
+    $phpfile = getenv('TEST_PHP_EXECUTABLE');
+    $impossiblefile = $phpfile.DIRECTORY_SEPARATOR.'bug28790.impossible';
+    $testfile = __DIR__.DIRECTORY_SEPARATOR.'bug28790.file';
+    $codefile = __DIR__.DIRECTORY_SEPARATOR.'bug28790.test.php';
+    $code = <<<EOC
+<?php
+
+function all_the_stats(\$filename, \$message) {
+    if (@lstat(\$filename)) {
+        print("lstat: \$message.\n");
+    }
+    if (@stat(\$filename)) {
+        print("stat: \$message.\n");
+    }
+}
+
+\$testfile = str_replace("\\\\", "/", "$testfile");
+passthru('$php -n -r \'touch("\$testfile");\'');
```

Comment:
`'` isn't a valid argument enclosure on Windows; only `"` is supported there.

Reply:
Ung.  OK, will fix in the "INI" suggestion you make below.

Reply:
Excellent.  Fixed, thanks.